pages/history.js

```diff
@@ -1,10 +1,92 @@
+import {
+	faBirthdayCake,
+	faCircleNotch,
+	faMask,
+	faRandom,
+	faVideo,
+} from "@fortawesome/free-solid-svg-icons";
 import Head from "next/head";
 import styled from "styled-components";
+import Event from "../components/event";
+
+const EventsContainer = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+
+	& > * {
+		flex: 0 1 calc(50% - 16px);
+		margin: 8px;
+
+		max-width: 35%;
+	}
+
+	@media (max-width: 800px) {
+		flex-direction: column;
+
+		& > * {
+			flex-basis: 100%;
+			max-width: unset;
+		}
+	}
+`;
 
 export default () => (
 	<>
 		<Head>
 			<title>History - The Snakeroom</title>
 		</Head>
+		<EventsContainer>
+			<Event icon={faMask} name="Imposter" time="April 2020">
+				In this event, players receive four messages by other players
+				and one from the bot, and must determine which one was generated
+				by the bot. The third iteration of the Sneknet collects results
+				from each round to determine which of the five messages was
+				generated by the bot. With enough data, the correct answer can
+				automatically be selected.
+			</Event>
+			<Event icon={faVideo} name="RPAN" time="August 2019">
+				RPAN is a streaming event with a cap on streamers and length.
+				The Snakeroom used a script that sniped slots and allowed users
+				to broadcast from an OBS stream, bypassing the intended
+				limitation of streaming from a mobile device and allowing
+				Snakeroom members to create <b>Place Live</b> and{" "}
+				<b>RPAN Plays Pokémon</b>. They were also able to stream prior
+				to the event going live, although Reddit admins quickly shut
+				this down.
```

Comment:
```suggestion
				The Snakeroom uses a script that snipes slots and allows users
				to broadcast from an OBS stream, bypassing the intended
				limitation of streaming from a mobile device and allowing
				Snakeroom members to create <b>Place Live</b> and{" "}
				<b>RPAN Plays Pokémon</b>. They are also able to stream prior
				to the event going live, although Reddit admins quickly shut
				this down.
```

Reply:
Resolved by instead changing to past tense in 74b3de0122571f988d2b292819c1e11bf2474a15